utils/publish-canary.js

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env node
+// @ts-check
```

Comment:
This helps illuminate the errors in this file. It doesn't seem to block any of our linter checks (which is good), just helpful.